build/template-run-device-auth-tests.yaml

```diff
@@ -10,5 +10,11 @@ steps:
 - task: VSTest@2
   displayName: 'DeviceAuthTests'
   inputs:
-    testAssemblyVer2: '$(System.ArtifactsDirectory)\drop\*\*'
+    testSelector: 'testAssemblies'
+    testAssemblyVer2: '$(System.ArtifactsDirectory)\drop\**\**'
+    searchFolder: '$(System.ArtifactsDirectory)'
+    rerunFailedTests: true
+    rerunMaxAttempts: '3'
+    runInParallel: false
+    codeCoverageEnabled: true
```

Comment:
nit: not much point in finding code coverage for integration tests... they are not meant to cover code (unit tests are), they are meant to verify integration points.